documentation/docs/extensibility.md

```diff
@@ -1,86 +1,199 @@
 # Extensibility
 
-There are several possibilities for extensibility besides the **[very powerful configuration](configuration.md)**:
+When using one of the ready-made pipelines project "Piper" provides, the basic idea is to not write custom pipeline code.
+The pipelines are centrally maintained, and can be used with a small amount of [configuration](configuration.md).
 
-## 1. Stage Exits
+For the large majority of _standard_ projects, the features of the ready-made pipelines should be enough to implement a production-ready CI/CD workflow with little effort.
 
-  You have to create a file like `<StageName>.groovy` (for example, `Acceptance.groovy`) and store it in folder `.pipeline/extensions/` in your source code repository.
+This approach allows you to focus on what you need to get done while implementing [Continuous Delivery](https://martinfowler.com/bliki/ContinuousDelivery.html) in a best-practice compliant way.
 
-!!! note "Cloud SDK Pipeline"
-    If you use the Cloud SDK Pipeline, the folder is named `pipeline/extensions/` (without the dot). For more information, please refer to [the Cloud SDK Pipeline documentation](https://github.com/SAP/cloud-s4-sdk-pipeline/blob/master/doc/pipeline/extensibility.md).
+If a feature you need is missing, or you discovered a bug in one of the ready-made pipelines, please see if there is already an [issue in our GitHub repository](https://github.com/SAP/jenkins-library/issues), and open a new one if that is not the case.
 
-  The pipeline template checks if such a file exists and executes it, if present.
-  A parameter that contains the following keys is passed to the extension:
+In some cases, specialised features might not be desirable for inclusion in the ready-made pipelines.
```

Comment:
Is there a general consensus regarding GB English versus American English spelling?

Reply:
🇺🇸 

Did I sneak in some 🍵 English here?

Reply:
Then it should be speciali*z*ed.